src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,16 +38,24 @@ public class RobotContainer {
    */
   public static Drive drive = new Drive();
 
+  /** mailboxPneumatics object for the MailboxPneumatics class */
+  public static MailboxPneumatics mailboxPneumatics = new MailboxPneumatics();
+
+  /** mailboxBelts object for the MailboxBelts class */
+  public static MailboxBelts mailboxBelts = new MailboxBelts();
+
   /*
    * ************
    * * COMMANDS *
    * ************
    */
 
   /* For now, we don't make commands public static, as there isn't really a reason to. */
```

Comment:
maybe remove this as it seems to no longer apply

Reply:
+1